olwebsite-app/src/contexts/UserContext.jsx

```diff
@@ -0,0 +1,45 @@
+import  React, { useContext, useEffect, useState } from 'react';
+
+const UserContext = React.createContext([undefined, undefined]);
+
+function UserProvider ({ children }) {
+    const [user, setUser] = useState(null);
+
+    useEffect(() => {
+        //TODO Jacob J: Don't use localStorage, very unsecure
+        const sessionUser = JSON.parse(window.localStorage.getItem('user'));
```

Comment:
to add to your TODO, I would tend to refrain from using magic numbers and strings like 'user'. Should probably refactor it out to a constant: userKey, or something similar

Reply:
I can certainly do that. This was just a temporary fix to get stuff running. Whatever system (probably cookies) that we implement to replace this, would get rid of this line though, so that's why I didn't do it upfront. I agree we should definitely use constants for similar situations though, and whatever we replace this with.